ui/packages/consul-ui/vendor/consul-ui/routes-debug.js

```diff
@@ -1,19 +1,20 @@
-(routes => routes({
-  ['oauth-provider-debug']: {
-    _options: {
-      path: '/oauth-provider-debug',
-      queryParams: {
-        redirect_uri: 'redirect_uri',
-        response_type: 'response_type',
-        scope: 'scope',
+(routes =>
+  routes({
+    ['oauth-provider-debug']: {
+      _options: {
+        path: '/oauth-provider-debug',
+        queryParams: {
+          redirect_uri: 'redirect_uri',
+          response_type: 'response_type',
+          scope: 'scope',
+        },
       },
-    }
-  },
-}))(
-  (json, data = document.currentScript.dataset) => {
-    const appNameJS = data.appName.split('-')
-      .map((item, i) => i ? `${item.substr(0, 1).toUpperCase()}${item.substr(1)}` : item)
-      .join('');
-    data[`${appNameJS}Routes`] = JSON.stringify(json);
+    },
+  }))(
+  (
+    json,
+    data = typeof document !== 'undefined' ? document.currentScript.dataset : module.exports
+  ) => {
+    data[`routes`] = JSON.stringify(json);
```

Comment:
This route is only complied/enabled if you are in development mode (note the routes-debug.js name of the file). It's the page/route for our fake OIDC provider we use for manually testing SSO.